KnowQuarter/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace KnowQuarter
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine("Введите координату точки X");
+            var h = double.Parse(Console.ReadLine());
+
+            Console.WriteLine("Введите координату точки Y");
+            var v = double.Parse(Console.ReadLine());
+
+            if (h>0 & v>0)
```

Comment:
Рекомендую использовать ленивую конъюнкцию (здесь и по возможности всегда):
```suggestion
            if (h>0 && v>0)
```